client/package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "firebase": "^6.3.5",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1"
+    "react-redux": "^7.1.0",
+    "react-router-dom": "^5.0.1",
+    "react-scripts": "3.1.1",
+    "recompose": "^0.30.0"
```

Comment:
What are we using recompose for in this instance? I'm pretty sure recompose is no longer being updated because hooks can do the things it was built for and I can look into how we can replace it with hooks.

Reply:
We may not need it. I used because I had multiple Higher order components that I wrapped the SignUpForm in during export.  Recompose, just makes sure that (for example) my withRouter HOC is applied before my fireBase HOC for that component. But I will have to check if that makes a difference. Yes, if hooks can do something similiar, we can replace it.